### PR TITLE
Question page

### PR DESCRIPTION
**Pull request by git-0r**
# Created questions page screen for quiz app
quiz4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,10 +6,82 @@
...
+                <circle cx="12" cy="10" r="3" />
+                <circle cx="12" cy="12" r="10" />
+            </svg>
+        </div>
+    </nav>
+
+    <main class="main-content">
+
+        <div class="quiz-container">
+            <p class="heading-xl">Sports Quiz</p>
```
Comment (manish-GH) [inline index.html]: This should probably be a header tag

File: style.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+}
+
+.navbar-input {
+  width: 80%;
+  height: 30px;
```
Comment (manish-GH) [inline style.css]: use rem

**Review by manish-GH — APPROVED** [review]
It's pretty similar to your rules page. If you are working on the responsiveness use min-height and flex. 
Other than that the UI is clean. Replace px with rem and consider using header tags wherever applicable, then you are good to merge

**Review by varun1233 — APPROVED** [review]
Code looks good to merge

File: question_page.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+.main-content {
+  width: 100%;
+  min-height: 90vh;
+  padding: 2rem;
+  background-color: var(--clr-primary-hover);
+}
+
+.quiz-container > * + * {
```
Comment (gouravmarch20) [inline question_page.css:26]: can you tell what is meaning of this

Comment (gouravmarch20) [reply question_page.css:26]: rest is fine

**Review by gouravmarch20 — APPROVED** [review]

File: question_page.css (hunk 3 above)
Comment (git-0r) [reply question_page.css:26]: http://alistapart.com/article/axiomatic-css-and-lobotomized-owls/

**Pull request merged**